### DISCUSSION TITLE
After last update lost Fab information -v176 (NET 4.8)

### DISCUSSION THREAD

**Discussion by R3d2q** [Rem0o/FanControl.Releases#2224] [General]
After last update v176 (NET 4.8) lost :
1. all fan given names, adjustments
2. Fan configurations
3. Fan min/max settings
4.
Can't run fan auto adjustment exclusively for fan
Can't get back fan settings from saved configurations

**Comment by Rem0o** [comment]
Where did you put the fancontrol files? Lookup in the configuration folder in there, your configs should still be there.

**Reply by R3d2q** [reply]
I see all config files. Loading, but nothing changes!
Made new config
